src/DynamoCoreWpf/ViewModels/Watch3D/Shaders/Custom.fx

```diff
@@ -139,13 +139,13 @@ PSInputCustom VShaderCustom(VSInputCustom input)
 	PSInputCustom output;
 
 	bool isSelected = input.customParams.x;
-	bool requiresPerVertexColoration = input.customParams.y;
+	bool requiresPerVertexColoration = input.customParams.z;
 
 	float4 inputp;
 	if (isSelected || requiresPerVertexColoration)
 	{
 		// Nudge the vertex out slightly along its normal.
-		float3 nudge = input.n * 0.0001;
+		float3 nudge = normalize(input.n) * 0.0001;
```

Comment:
@jnealb This issue is not meant to be addressed in this PR, but could you explain what would be the expected CORRECT behavior here?

In this case, the `Display` is rendered (virtually) *in front* of the `Sphere` geometry, hence it can always be seen by the camera.

![red-sphere](https://cloud.githubusercontent.com/assets/6386550/23451532/40ba2242-fe9b-11e6-9878-45a74f233645.gif)

A different story will happen when the `Display`'s alpha is not 255, then it will be rendered as *behind* the `Sphere`.